brokerapi/brokers/cloudsql/common-definition.go

```diff
@@ -84,25 +84,25 @@ var commonProvisionVariables = []broker.BrokerVariable{
 	{
 		FieldName: "disk_type",
 		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances) ",
+		Details:   "(only for 2nd generation instances)",
```

Comment:
Should this have an enum field as well?

Reply:
I need to investigate this one further, probably as part of #155 and #254. When I was looking at the docs, "ssd" wasn't even listed as a valid type, so I need to investigate what's going on there.